### DISCUSSION TITLE
启动失败，获取启动参数失败是怎么回事该怎么解决

### DISCUSSION THREAD

**Discussion by lxr631** [Meloong-Git/PCL#4239] [提问] [answered]
![bandicam 2024-07-12 17-40-27-253 (1)](https://github.com/user-attachments/assets/e1ba5cd7-673a-4b08-a627-3d5c2d1da481)

**Comment by allMagicNB** [comment, marked as answer]
右键以管理员身份运行再试试

**Reply by WForst-Breeze** [reply]
参阅 https://github.com/Hex-Dragon/PCL2/issues/4243#issuecomment-2226796191

**Comment by Sodium2He** [comment]
右键以管理员身份运行应该可以
这个问题在issues中已有提及并解决
